diff -urN kdenetwork-4.10.1/krfb/krfb/CMakeLists.txt kdenetwork-4.10.1-patched/krfb/krfb/CMakeLists.txt
--- kdenetwork-4.10.1/krfb/krfb/CMakeLists.txt	2013-03-01 16:21:49.705336890 +1100
+++ kdenetwork-4.10.1-patched/krfb/krfb/CMakeLists.txt	2013-03-11 01:33:39.454420962 +1100
@@ -24,7 +24,7 @@
 
 set_target_properties (krfbprivate PROPERTIES
                        VERSION ${GENERIC_LIB_VERSION}
-                       SOVERSION ${GENERIC_LIB_VERSION}
+                       SOVERSION ${GENERIC_LIB_SOVERSION}
 )
 
 install (TARGETS krfbprivate
